Marlin/Marlin_main.cpp

```diff
@@ -1691,6 +1691,11 @@ static void do_blocking_move_to(float x, float y, float z, float feed_rate = 0.0
   feedrate = old_feedrate;
 }
 
+inline void do_blocking_move_to_axis(AxisEnum axis, float where, float feed_rate = 0.0) {
```

Comment:
> Wouldn't it make sense to be do_blocking_axis_move_to() ?

No, not to me.
_to() cold be _to_xyz()
_to_x()
_to_y()
_to_xy()
_to_z()
_to_axis()